src/main/java/org/fundacionjala/coding/manuel/Kata5.java

```diff
@@ -0,0 +1,7 @@
+package org.fundacionjala.coding.manuel;
+
+/**
+ * Created by Administrator on 3/5/2018.
+ */
+public class Kata5 {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty class, write its code or make it an "interface". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2094)